apps/nowcasting-app/components/utils.ts

```diff
@@ -51,3 +51,8 @@ export const formatISODateStringHuman = (date: string) => {
   // further formatting could be done to make it yyyy/mm/dd HH:MM
   return `${date_london} ${date_london_time}`;
 };
+export const addMinutesToISODate = (date: string, munites: number) => {
+  var d = new Date(date);
+  d.setMinutes(d.getMinutes() + munites);
+  return d.toISOString();
```

Comment:
Timezone here is zero UTC not like London timezone

Reply:
that's ok because we use utc for all data , and only convert it for displayed time and dates